rskj-core/src/test/java/co/rsk/validators/BlockTimeStampValidationRuleTest.java

```diff
@@ -64,4 +64,54 @@ public void blockInTheFuture() {
         Assert.assertFalse(validationRule.isValid(block));
     }
 
+    @Test
+    public void blockTimeLowerThanParentTime() {
+        int validPeriod = 540;
+        BlockTimeStampValidationRule validationRule = new BlockTimeStampValidationRule(validPeriod);
+
+        Block block = Mockito.mock(Block.class);
+        Block parent = Mockito.mock(Block.class);
+
+        Mockito.when(block.getTimestamp())
+                .thenReturn(System.currentTimeMillis() / 1000);
```

Comment:
These tests are flaky because they depend on the system clock. For example, if for some reason (e.g. garbage collection) calling `validationRule.isValid` takes more than a second the test will fail.

Maybe add a `TODO` or tracking issue (this is **super low-prio**).